ntc_templates/templates/alcatel_aos_show_vlan_port.textfsm

```diff
@@ -0,0 +1,12 @@
+Value VLAN_ID (\d+)
+Value PORT ((\d+\/)?\d+\/\d+)
```

Comment:
```suggestion
Value PORT ((\d+\/?)+)
```

This supports `1/1` and `1/1/1` ... plus more.

We can shorten this pattern quite a bit and even have the potential support `1/1/1/1` and beyond.
(Not that an interface would be named that anytime soon.)

Reply:
Also added this new suggestion :)

I doubt in the near future will see 1/1/1/1/1/1 but who knows xd